### ISSUE TITLE
Custom registries are broken

### ISSUE THREAD

**Issue by thorn0** [mjeanroy/bower-npm-resolver#253]
Works with 0.9.1, but 0.10.0 gives me this:

```
>> Cleaned target dir [...]\vendor
{
  headers: {
   ...
  },
  statusCode: 404,
  code: 'E404',
  method: 'GET',
  uri: 'https://registry.npmjs.org/@fortawesome%2ffontawesome-pro',
  body: { error: 'Not found' },
  pkgid: '@fortawesome/fontawesome-pro@5.10.2',
  message: '404 Not Found - GET https://registry.npmjs.org/@fortawesome%2ffontawesome-pro - Not found'
}
Fatal error: 404 Not Found - GET https://registry.npmjs.org/@fortawesome%2ffontawesome-pro - Not found
```

**Comment by mjeanroy** [comment]
@thorn0 Sorry for the (very long) delay.

How is your custom registry configured?

**Comment by thorn0** [comment]
No worries. I understand, it's open source. :)

So, I have the `.npmrc` file in the same folder where `bower.json` is:
```
@fortawesome:registry=https://npm.fontawesome.com/
//npm.fontawesome.com/:_authToken=[...]
```
OS: Windows 10 x64

**Comment by mjeanroy** [comment]
Thanks.

You probably have already tested, but does it work if the `.npmrc` file is located in your home folder (i.e the registry is defined "globally", not per project)?

**Comment by thorn0** [comment]
> does it work if the .npmrc file is located in your home folder

It doesn't.

> Are your bower.json and package.json files located at the root of your project?

They're in the same directory, but it's not the root of the project. `.git` is a few directories higher, however there is no other `package.json` files up the tree.

**Comment by thorn0** [comment]
But I tried to isolate this issue. Started in an empty folder from scratch... and it works. It's indeed something specific to my project.

**Comment by mjeanroy** [comment]
Hi @thorn0,
I just pushed a fix on the master branch, can you try and tell me if it fixes your issue?
A simple way to test it is to checkout the repo and link it in your project:

```
git checkout git@github.com:mjeanroy/bower-npm-resolver.git
cd bower-npm-resolver
npm install && npm run build && npm link .
```

Then in your project:

```
npm link bower-npm-resolver
```

Sorry for the delay, I had a lot of work and it is hard to debug... :/

**Comment by thorn0** [comment]
It works. 🎉 Thanks.

**Comment by mjeanroy** [comment]
Hi @thorn0,
I just published version 0.11.0 with the fix for custom registries. I also made some internal changes to ensure this resolver will be compatible with npm 7 (I made some tests with npm 7-beta7).
Feel free to open an issue if something does not work as expected!
